src/components/App/App.tsx

```diff
@@ -1,19 +1,53 @@
-import React, { useState } from "react";
+import React, { useEffect } from "react";
 
-import { Route, Routes } from "react-router-dom";
+import { PageRoutes } from "src/constants/routeNames";
+import { LocalStorageKey } from "src/enums/localStorageEnum";
+
+import { UseActions } from "src/hooks/useActions";
+import { useAppSelector } from "src/hooks/useAppSelector";
+
+import { IRoute } from "src/interfaces/IRoute";
 
 import Login from "src/pages/Login/Login";
+import MainPage from "src/pages/MainPage/MainPage";
 import Registration from "src/pages/Registration/Registration";
 
+import { RoutePath } from "../RoutePath/RoutePath";
+
 import "./App.css";
 
 const App = () => {
+  const { isAuth } = useAppSelector((state) => state.user);
+  const { setUserData } = UseActions();
+
+  useEffect(() => {
+    const User = localStorage.getItem(LocalStorageKey.authKey);
```

Comment:
Please do not use capital letters for variables

Reply:
Done.